Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM ubuntu:16.04
```

Comment:
any reason alpine will not work here? - https://github.com/docker-library/php/blob/e40afc3c7605f78cf2c5c0a1c419f9cf849d07c9/7.2-rc/alpine/Dockerfile and add nginx on the official php alpine image?

Reply:
or addd php to nginx alpine : https://github.com/nginxinc/docker-nginx/blob/14aa3b1b80341099afbf90eb0a9b9061b7145f18/mainline/alpine/Dockerfile

Reply:
Tried with both 7.0 and 7.2 fpm alpine in the library but both fails on build :) need more debug.